demo/tests/perfs/test_perfs_rec.perf.yml

```diff
@@ -165,16 +161,16 @@ TestPerfRec.update-form-with-changes:
 - db: 'UPDATE "formidable_access" SET ... WHERE "formidable_access"."id" = #'
 - db: 'UPDATE "formidable_access" SET ... WHERE "formidable_access"."id" = #'
 - db: 'UPDATE "formidable_access" SET ... WHERE "formidable_access"."id" = #'
-- db: RELEASE SAVEPOINT `#`
 - db: 'SELECT ... FROM "formidable_field" WHERE "formidable_field"."form_id" = # ORDER BY "formidable_field"."order" ASC'
 - db: SELECT ... FROM "formidable_item" WHERE "formidable_item"."field_id" IN (...) ORDER BY "formidable_item"."order" ASC
 - db: SELECT ... FROM "formidable_default" WHERE "formidable_default"."field_id" IN (...)
 - db: SELECT ... FROM "formidable_validation" WHERE "formidable_validation"."field_id" IN (...)
 - db: SELECT ... FROM "formidable_access" WHERE "formidable_access"."field_id" IN (...)
+- db: RELEASE SAVEPOINT `#`
 TestPerfRec.update-form-with-changes-pg:
 - db: 'SELECT ... FROM "django_session" WHERE ("django_session"."expire_date" > #::timestamptz AND "django_session"."session_key" = #)'
-- db: 'SELECT ... FROM "formidable_formidable" WHERE "formidable_formidable"."id" = #'
 - db: SAVEPOINT `#`
+- db: 'SELECT ... FROM "formidable_formidable" WHERE "formidable_formidable"."id" = # FOR UPDATE'
```

Comment:
all the tests suffixed by `-pg` are PG-only. as you can see, the first query after the start of the transaction is the "SELECT ... FOR UPDATE"